README.md

```diff
@@ -5,21 +5,17 @@ Rust-Lightning
 [![Documentation](https://img.shields.io/static/v1?logo=read-the-docs&label=docs.rs&message=lightning&color=informational)](https://docs.rs/lightning/)
 [![Safety Dance](https://img.shields.io/badge/unsafe-forbidden-success.svg)](https://github.com/rust-secure-code/safety-dance/)
 
-`rust-lightning` is a Bitcoin Lightning library written in Rust. The main crate,
-`lightning`, does not handle networking, persistence, or any other I/O. Thus,
-it is runtime-agnostic, but users must implement basic networking logic, chain
-interactions, and disk storage. More information is available in the `About`
-section.
+[LDK](lightningdevkit.org)/`rust-lightning` is a highly performant implementation of the Lightning
+Network protocol, with support for custom data persistence, chain sync, and much more.
+
+The main crate, `lightning`, does not handle networking, persistence, or any other I/O. Thus, it is
+runtime-agnostic, but users must plug in networking logic, chain interactions, and disk storage.
+More information is available in the [`About`](./README.md#about) section.
 
 Status
 ------
-The project implements all of the [BOLT
-specifications](https://github.com/lightning/bolts). The
-implementation has pretty good test coverage that is expected to continue to
-improve. It is also anticipated that as developers begin using the API, the
-lessons from that will result in changes to the API, so any developer using this
-API at this stage should be prepared to embrace that. The current state is
-sufficient for a developer or project to experiment with it.
+The project implements all of the [BOLT specifications](https://github.com/lightning/bolts),
+and has been in production use since 2021.
```

Comment:
"We would still recommend to be very careful and extremely conservative with deploying LDK in production. The whole protocol Lightning itself could be considered as "beta", with numerous security or safety flaws known. Deployment for now should favor connections to trusted or semi-trusted peers".

I know, it's opinionated words, though we should be mindful in what we say to our users I think.

Reply:
Well we can reformulate as something like "We would recommend to be conservative and careful by deploying LDK in production. The protocol Lightning itself can still be considered as "beta" with numerous security/safety flaws known affecting _all_ the implementations. Yet, LDK has still been historically the most robust one". (well Eclair could contest that claim).

That's the thing marketing doesn't marry well with sound security advices :thinking:

Reply:
Let's do something less opinionated like "...and has been in production use since 2021. We would still recommend to be conservative and careful by deploying LDK in production. From the majority of Lightning developers opinions, the protocol itself is still to be considered in "beta". Wdyt ?

Reply:
I think it's a given for any implementation that care is needed in prod deployment, so I still see that copy as putting LDK at a disadvantage compared to other LN implementations' readmes. So I'm still in favor of sticking with the current version or similar and having a nuanced security discussion in a different document. But would like to move this along so I'll go ahead and add something like that if you feel strongly 😛

Reply:
Lmk what you think of the new copy

Reply:
Okay I'm fine with the current version and I feel it's on me to add a nuanced security discussion in a different document :)